lib/pyspark_matrix_similarity.py

```diff
@@ -0,0 +1,30 @@
+from pyspark.sql import DataFrame
+from lib.pyspark_cosinus_similarity import cosine_similarity
+import pandas as pd
+
+
+def cosine_similarity_for_row(
+    df: DataFrame,
+    movie_id: str,
+) -> DataFrame:
```

Comment:
Skoro ma zwracać pandasowy DataFrame to powinno być pd.DataFrame, bo teraz odwołujesz się do DataFrame z pyspark.sql

Reply:
Zmieniony na sparkowy, już jest ok